src/server/wesnothd/server.hpp

```diff
@@ -192,20 +193,16 @@ class server : public server_base
 
 	player_connections player_connections_;
 
-	std::deque<std::shared_ptr<game>> games() const
+	std::map<int, std::shared_ptr<game>> games() const
```

Comment:
What is this for?

Reply:
Probably left over from when I thought I needed the id. The deque use does seem like a weird design though.

Reply:
Given the implementation it seems sensible? (Though the implementation seems more complicated than necessary so maybe a vector could be used instead)

You wouldn't use this function to find a game by ID. You would check `player_connections_`. I suppose there could be some more helper functions to use `player_connections` since it always takes a bit to wrap your head around the multi-index stuff.

Reply:
That would definitely be helpful. If not for the usage added in this PR, `games()` wouldn't even need to return any type of container, it could just return an int since the existing usage is only checking its size or if it's empty.